Dockerfile-csi-controller

```diff
@@ -12,23 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM centos:7
-RUN yum --enablerepo=extras -y install epel-release && yum -y install python36-pip
+FROM registry.access.redhat.com/ubi8/python-36:1
+MAINTAINER IBM Systems Engineering <TBD>
```

Comment:
How about "IBM Systems Storage"? but I think it's better to ask Shaluka/Matt if it's customer facing